src/lib.rs

```diff
@@ -236,6 +236,9 @@ compile_error!(
     `moka` crate. Please update your dependencies in Cargo.toml"
 );
 
+// Reexport(s)
+pub use equivalent;
```

Comment:
Perhaps, we should reexport just a trait rather than the entire module (crate)?

```suggestion
pub use equivalent::Equivalent;
```